lms/djangoapps/discussion/tasks.py

```diff
@@ -82,6 +90,31 @@ def send_ace_message(context):  # lint-amnesty, pylint: disable=missing-function
             _track_notification_sent(message, context)
 
 
+@shared_task(base=LoggedTask)
+@set_code_owner_attribute
+def send_ace_message_for_reported_content(context):  # lint-amnesty, pylint: disable=missing-function-docstring
+    context['course_id'] = CourseKey.from_string(context['course_id'])
+    context['course_name'] = modulestore().get_course(context['course_id']).display_name
+
+    moderators = get_users_with_moderator_roles(context)
+    context['site'] = Site.objects.get(id=context['site_id']
+                                       )
+    if not _is_content_still_reported(context):
+        log.info('Reported content is no longer in reported state. Email to moderators will not be sent.')
```

Comment:
This log does not seem useful. I think if we can add details of the specific content it would be useful

Reply:
going to address this is the followup ticket!